dotCMS/src/main/java/com/dotmarketing/portlets/workflows/util/WorkflowImportExportUtil.java

```diff
@@ -116,64 +144,58 @@ public void importWorkflowExport(File file) throws IOException {
 				dc.loadResult();
 			}
 
-			wapi.saveWorkflowActionClassParameters(importer.getActionClassParams());
+			workflowAPI.saveWorkflowActionClassParameters(importer.getActionClassParams());
 
 		} catch (Exception e) {// Catch exception if any
 			Logger.error(this.getClass(), "Error: " + e.getMessage(), e);
 		} finally {
 
-			in.close();
+			CloseUtils.closeQuietly(bufferedReader);
+		}
+	}
+
+	private User getCurrentUser () {
 
+		User user = null;
+
+		final HttpServletRequest request =
+				HttpServletRequestThreadLocal.INSTANCE.getRequest();
+
+		if (null != request) {
+
+			try {
+				user = PortalUtil.getUser(request);
+			} catch (PortalException | SystemException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)